src/content/docs/en/tutorial/4-layouts/1.mdx

```diff
@@ -139,7 +139,7 @@ You still have some Astro components repeatedly rendered on every page. It's tim
 
 ## Try it yourself - Use your layout everywhere
 
-**Refactor** your other pages (`blog.astro` and `about.astro`) so that they use your new `<BaseLayout>` component to render the common page elements.
+**Refactor** your other pages (`blog.astro` and `about.astro`) so that they use your new `<BaseLayout>` component to render the common page elements. (Note: using the BaseLayout to render your `about.astro` page means you will lose the style for h1. If you want to keep the h1 style, you need to pass the h1 style to BaseLayout as props via component attribute.)
```

Comment:
```suggestion
**Refactor** your other pages (`blog.astro` and `about.astro`) so that they use your new `<BaseLayout>` component to render the common page elements. (Note: using the BaseLayout to render your `about.astro` page means you will lose the `<style>` tag added to the `<head>` of this page. If you want to keep the custom `<h1>` style, move the style tag to the body of the page component.)
```
This is probably the solution I'd give at this stage in the tutorial! It is a simpler step, and also demonstrates that this style tag can live in the body of the component.